web/src/main/webapp/css/classic/main.css

```diff
@@ -11,7 +11,6 @@
 
 BODY {
 	font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
-	font-size: 75%;
```

Comment:
Heh, I'm so used to the small fonts, that just looks gigantic.

I'm a bit concerned how it will look on narrower monitors. I think we're well past the era of 1024x768, but 1440w isn't exactly embarrassing. How does it look in a smaller window?

The vertical alignment of icons looks off. If people think the interface needs work, we should consider an overhaul, maybe using Bootstrap or something.